File: expressions/budget.c
#include <stdio.h>

int main(void){
    float income, rent, utilities, groceries, transportation, savings, expenses, spend;
    float prent, putilities, pgorceries, ptransportation, psavings, pexpenses
    printf("This is a budget calculator. How much do you make a month?\n");
    scanf("%f", &income);
    printf("How much does your rent cost?\n");
    scanf("%f", &rent);
    printf("How much does your utilities cost?\n");
    scanf("%f", &utilities);
    printf("How much do your groceries cost?\n");
    scanf("%f", &groceries);
    printf("How much does your transportation cost?\n");
    scanf("%f", &transportation);
    expenses = rent + utilities + groceries + transportation;
    savings = income * .2;
    spend = income - expenses - savings;
    printf("You make %.2f\n", &income);
    printf("Your expenses are %.2f\n", &expenses);
    printf("Your savings are %.2f\n", &savings);
    printf("Your spending money is %.2f\n", &spend);
    prent = rent/income * 100;
    putilities = utilities/income * 100;
    ptransportation = transportation/income * 100;
    psavings = savings/income * 100;
    pexpenses = expenses/income * 100;
    print("your rent is %d%%" of your income.\n, (int) prent);
    print("your utilities is %d%%" of your income.\n, (int) putilities);
    print("your transportation is %d%%" of your income.\n, (int) ptransportation);
    return 0;
}

File: expressions/budget.py
income = float(input("This is a budget calculator. How much do you make a month?\n"))
rent = float(input("How much does your rent cost?\n"))
utilities = float(input("How much does your utilities cost?\n"))
groceries = float(input("How much do your groceries cost?\n"))
transportation = float(input("How much does your transportation cost?\n"))

expenses = rent + utilities + groceries + transportation
savings = income * 0.2
spend = income - expenses - savings

print(f"You make {income:.2f}")
print(f"Your expenses are {expenses:.2f}")
print(f"Your savings are {savings:.2f}")
print(f"Your spending money is {spend:.2f}")

prent = (rent / income) * 100
putilities = (utilities / income) * 100
ptransportation = (transportation / income) * 100
psavings = (savings / income) * 100
pexpenses = (expenses / income) * 100

print(f"Your rent is {int(prent)}% of your income.")
print(f"Your utilities is {int(putilities)}% of your income.")
print(f"Your transportation is {int(ptransportation)}% of your income.")

